src/watchful.ts

```diff
@@ -86,15 +93,18 @@ export class Watchful extends Construct implements IWatchful {
       );
     }
 
-    this.dash = new cloudwatch.Dashboard(this, 'Dashboard', { dashboardName: props.dashboardName });
+    if (!props.noDashboard){
```

Comment:
should also trigger if dashboardName is provided (requires a doc update)

Reply:
Changed condition to:
```
const dashboard = props.dashboard ?? true;
if (dashboard || props.dashboardName){
```
and updated the docs.